features/having_credit_memo_generated.feature

```diff
@@ -52,6 +54,8 @@ Feature: Having credit memo generated
         And it should have sequential number generated from current date
         Then this credit memo should contain 1 "Mr. Meeseeks T-Shirt" product with "$0.50" tax applied
         And it should be issued in "United States" channel
+        And its subtotal should be "$5.00"
+        And it should have a tax item "US VAT (10%)" with amount "$0.50"
```

Comment:
It would be amazing to have a test also for the PDF generated 🌮 but this is definitely for the future